src/traces/bar/attributes.js

```diff
@@ -39,6 +39,51 @@ var marker = extendFlat({
         max: 1,
         editType: 'style',
         description: 'Sets the opacity of the bars.'
+    },
+    patternfill: {
```

Comment:
maybe just `pattern`?

Reply:
Fixed in 39e35ccc02eb5d76ea7bbdc88be61a7cb1943918.